Runtime/Random/RandomStringImpl.cs

```diff
@@ -0,0 +1,73 @@
+// Copyright (c) 2023 Koji Hasegawa.
+// This software is released under the MIT License.
+
+using System;
+using System.Text;
+using UnityEngine.Assertions;
+
+namespace TestHelper.Monkey.Random
+{
+    public class RandomStringImpl : IRandomString
+    {
+        internal const string CharsASCIIDigits = "0123456789";
+        internal const string CharsASCIILowers = "abcdefghijklmnopqrstuvwxyz";
+        internal const string CharsASCIIUppers = "ABCDEFGHIJKLMNOPQRSTUVWXYZ";
+        internal const string CharsASCIISymbols = "!\"#$%&'()*+,-./:;<=>?@[\\]^_`{|}~";
+        internal const string CharsASCIIAlphanumeric = CharsASCIIDigits + CharsASCIILowers + CharsASCIIUppers;
+        internal const string CharsASCIIPrintable = CharsASCIIAlphanumeric + CharsASCIISymbols;
+
+        private static StringBuilder _sb = new StringBuilder();
+        private readonly IRandom _random;
+
+
+        /// <summary>
+        /// Initializes a new instance of the <c>RandomStringImpl</c> class,
+        /// using the specified random number generator.
+        /// <remarks>This class is not thread-safe.</remarks>
+        /// </summary>
+        /// <param name="random">Random number generator</param>
+        public RandomStringImpl(IRandom random)
+        {
+            _random = random;
+        }
+
+
+        /// <summary>
+        /// Returns a random string.
+        /// <remarks>This method is not thread safe.</remarks>
+        /// </summary>
+        /// <param name="parameters">String generation parameters</param>
+        /// <returns>Generated string</returns>
+        public string Next(RandomStringParameters parameters)
```

Comment:
IMO:
Has only three parameters, so I think it is okay to receive it directly as a parameter.

Reply:
I decided this choice because I felt `Func<GameObject, RandomStringParameters>` was better than `Func<GameObject, (int, int, CharacterKind)>`. If you don't like the struct, I'm gonna extract these parameters. How do you feel it?

Reply:
You're right. So keep RandomStringParameters, please.